webpack/assets/javascripts/react_app/API.js

```diff
@@ -18,7 +18,7 @@ export default {
       data: data,
       error: function (jqXHR, textStatus, errorThrown) {
         /* eslint-disable no-console */
-        console.log(jqXHR);
+        // console.log(jqXHR);
```

Comment:
?

Reply:
I'll uncomment this - not sure what the comments are for. They were obscuring my test results.